### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by robthebank** [avocode/avocode#1923]
---
* **Version**: 2.21.0
* **Sketch plugin version**: 48.2.8
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 287148
---
```
Error: Could not download metadata (revision=21243935)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1303374
    at <anonymous>
```

**Comment by jakubzitny** [comment]
Hi @robthebank, thanks for the report. We have some problems with rendering your file in our backend, we're looking into it.

**Comment by jakubzitny** [comment]
Hi, this problem should be released in latest Avocode 2.22.0. Please update and let us know if it's still happening.
